operations/rack_resiliency/rr_kyverno.md

```diff
@@ -0,0 +1,33 @@
+
+## Spreading critical services across racks with `kyverno`
+
+Some of the CSM critical services already have established the pod affinities to spread the replicas across nodes. Because the nodes which are picked
+by the `Kubernetes` scheduler can be on the same rack, it is necessary to include a topology constraint for these services; this helps the `Kubernetes` scheduler 
+distribute the replicas across zones. This is achieved using the "Pod Topology Spread Constraints" feature.
```

Comment:
Is this a Kubernetes feature? If so, probably we should say that explicitly, and say something like `For more information, see [the Kubernetes documentation](link-to-the-docs)`. Unless this is a feature we talk about elsewhere in the CSM docs, in which case we could link there.